.changeset/unlucky-carpets-dance.md

```diff
@@ -0,0 +1,8 @@
+---
+"@frameless/pdc-dashboard": minor
+"@frameless/vth-dashboard": minor
+"@frameless/strapi-tiptap-editor": minor
+---
+
+- Toegevoegd: Nederlandse vertalingen voor TipTap editor extensies. Bij selectie van de Nederlandse taal in het Strapi-dashboard wordt de toolbar in het Nederlands weergegeven.
+- Toegevoegd: Focusfunctionaliteit voor de editor table-extensie in het Strapi-dashboard. Bij het invoegen van een tabel kan de muiscursor eenvoudig boven en onder de tabel worden verplaatst. Bij hover wordt een border en twee invoegknoppen weergegeven. Meer details in [de handleiding](linkhttps://vth-api.cg-intern.acc.utrecht.nl/manual/index.html).
```

Comment:
"Focusfunctionaliteit voor de editor table-extensie in het Strapi-dashboard." is vrij technisch.

Wat als je gelijk de doelgroep aanspreekt? Bijvoorbeeld:

Wanneer je een tabel maakt in Strapi, is het nu makkelijker om tekst voor of na de tabel toe te voegen. Klik op de knop boven de tabel om daar een lege regel toe te voegen, of klik op de knop onder de tabel om erna een lege regel te maken.